### DISCUSSION TITLE
I have been made an "owner" of the organization, but it still shows that I'm a "member"

### DISCUSSION THREAD

**Discussion by ymarynych** [community/community#111318] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I need an "owner"-permission of the organization I'm in. So I asked one of the "owners" to give me such permissions. He did that, I received an email saying "You have been made an owner of the ****** organization", but when I go to People-tab I'm still in "members"-group, but not in "owners". 
Any suggestions are welcome.
Thank you

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ymarynych ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by ymarynych** [comment, marked as answer]
I'm closing out the ticket. The issue was at my end. Other person who is Owner made me Owner, but then reverted back my permissions. That's why I saw I was still Member.
